Kudu.Contracts/Kudu.Contracts.csproj

```diff
@@ -142,6 +142,7 @@
     <Compile Include="Settings\DeploymentSettingsExtension.cs" />
     <Compile Include="Settings\IDeploymentSettingsManager.cs" />
     <Compile Include="Settings\SettingsKeys.cs" />
+    <Compile Include="SiteExtensions\InvalidEndpointException.cs" />
```

Comment:
in update scenario, i do catch this exception and send back specify error message to caller, to let request initiator realize the feed endpoint is invalid.

if we only log it, we might get a live site ticket, and will only find out from log.